src/main/java/pl/bykowsi/kurs/tydzien1pd/controller/impl/ShopControllerPlus.java

```diff
@@ -0,0 +1,28 @@
+package pl.bykowsi.kurs.tydzien1pd.controller.impl;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.context.event.ApplicationReadyEvent;
+import org.springframework.context.annotation.Profile;
+import org.springframework.context.event.EventListener;
+import org.springframework.stereotype.Controller;
+import pl.bykowsi.kurs.tydzien1pd.controller.ShopController;
+import pl.bykowsi.kurs.tydzien1pd.service.ShopService;
+
+@Controller
+@Profile("plus")
+public class ShopControllerPlus implements ShopController {
```

Comment:
jeśli uznasz, że zmieniasz na `ShopInspector` to w klasach implementujących też pozmieniaj